src/audio/aria/aria.c

```diff
@@ -162,9 +162,9 @@ static void aria_set_stream_params(struct comp_buffer *buffer,
 	const struct ipc4_audio_format *audio_fmt = &mod->priv.cfg.base_cfg.audio_fmt;
 
 	ipc4_update_buffer_format(buffer, audio_fmt);
-#ifdef ARIA_HIFI3
+#if SOF_USE_HIFI(3, ARIA) || SOF_USE_HIFI(4, ARIA)
```

Comment:
patch1 can be squashed into patch2?

Reply:
I don't think it is a good idea, patch 1 adds the align limit for hifi5, patch 2 is for new style, they are totally different things